blocks/library/quote/index.js

```diff
@@ -20,11 +20,21 @@ registerBlock( 'core/quote', {
 		citation: html( 'footer' )
 	},
 
+	controls: [ '1', '2' ].map( ( level ) => ( {
+		icon: 'format-quote',
+		title: wp.i18n.sprintf( wp.i18n.__( 'Quote %s' ), level ),
```

Comment:
We should probably use `%d` placeholder, see also https://github.com/WordPress/gutenberg/pull/423#discussion_r112088447